static/js/util/ProfileFormFields.js

```diff
@@ -64,13 +66,41 @@ export default class ProfileFormFields extends React.Component {
     }));
   }
 
+  stepBack = () => {
+    this.context.router.push(this.prevUrl);
+  };
+
   saveAndContinue = () => {
-    let lastTab = this.nextUrl === "/dashboard";
-    saveAndContinue.call(this, lastTab).then(() => {
+    saveProfileStep.call(this, this.isLastTab).then(() => {
       this.context.router.push(this.nextUrl);
     });
   };
 
+  progressControls = (cellWidth=10) => {
+    let prevButton, nextButton;
+    if(this.prevUrl) {
+      prevButton = <Button
+        raised
+        className="progress-button previous"
+        onClick={this.stepBack}>
+        <span>Previous</span>
+      </Button>;
+    }
+    if(this.nextUrl) {
+      nextButton = <Button
+        raised
+        colored
+        className="progress-button next"
+        onClick={this.saveAndContinue}>
+        <span>{this.isLastTab ? "I'm Done!" : "Save and Continue"}</span>
+      </Button>;
+    }
+    return <Cell col={cellWidth}>
+        {prevButton}
+        {nextButton}
+      </Cell>;
```

Comment:
What do you think about this function just returning something like:

```js
<div>
  {prevButton}
  {nextButton}
</div>
```

and then leaving the responsibility of wrapping the buttons in a `<Cell>` component to the calling context? I think the default argument and so on is a little bit of indirection, when we could just have the information right there where the function is called at render time. If we have this JSX on, for instance, the 'professional' tab, it's a little clearer when reading the code what will actually render on the screen:

```js
<Cell col={10}>
  {this.progressControls()}
</Cell>
```
whereas if we have

```js
{this.progressControls(12)}
```

I have to go read the definiton of `progressControls` to figure out what that argument is going to do.